checks/src/test/resources/ru/hh/checkstyle/WrapNewLinesMethodChainCallCheckTestInput.java

```diff
@@ -59,7 +59,7 @@ public static void main(String[] args) {
         .stream()
         .collect(Collectors.toSet());
 
-    //expression with static method - NOT OK
+    //expression with static method - NOT OK or OK depending on "allowUnwrappedFirstCall" property
```

Comment:
```suggestion
    //expression with static method - NOT OK but firstCall is OK depending on "allowUnwrappedFirstCall" property
```

Предлагаю также как и ниже, ведь сам violation всё-таки должен возникнуть